packages/core/src/Cardano/types/Transaction.ts

```diff
@@ -82,9 +81,29 @@ export interface Redeemer {
 
 export type Signatures = Map<Ed25519PublicKey, Ed25519Signature>;
 
-export type Witness = Omit<Partial<Ogmios.TxAlonzo['witness']>, 'redeemers' | 'signatures'> & {
+export declare type Signature = string;
+export declare type ChainCode = string;
```

Comment:
Let's find what this is exactly, maybe we can better type it

Reply:
I couldn't find much information about it, other than it is a byte array, I will try tomorrow again.

Reply:
Could you please just make a list of types to be narrowed down and create a separate ticket for it?

Reply:
ticket created (ADP-2146), as far as I can see only those two types remaining unidentified.